src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -191,122 +155,141 @@ public String  MakeGUID() {
 
     public void BeginConnectPeer(String toPeerId, ConnectStatusCallback connectStatusCallback) {
         ConnectStatusCallback tmpCallback = connectStatusCallback;
-
-        if(tmpCallback == null) {
+        if (tmpCallback == null) {
```

Comment:
Shouldn't this literally never happen and therefore should cause an exception to be thrown?